src/jitter.rs

```diff
@@ -693,6 +693,11 @@ fn get_nstime() -> u64 {
     t as u64
 }
 
+#[cfg(all(target_arch = "wasm32", not(target_os = "emscripten")))]
+fn get_nstime() -> u64 {
+    panic!("Not supported")

```

Comment:
Could you modify `test_timer` to return an appropriate `TimerError` instead of panicking?

I guess this impl needs to stay in any case (for linking); shame but I guess we have to live with redundant error code.